### PR TITLE
[webui] minor change in Image Template

### PR DESCRIPTION
**Pull request by DavidKang**
Now the radio button is checked in the first package of the first
project with packages

Before:
screenshot before

After:
screenshot after

### CODE REVIEW

File: src/api/app/views/webui/image_templates/index.html.haml [hunk 1]
```diff
@@ -25,7 +25,9 @@
                   = template.description
         - if project.packages.empty?
           %p
-            There are currently no base image templates available for project <strong>#{title_or_name(project)}</strong>.
+            There are currently no base image templates available for project
+            = succeed '.' do
```
Comment (ChrisBr) [inline src/api/app/views/webui/image_templates/index.html.haml]: What does this do?

Comment (DavidKang) [inline src/api/app/views/webui/image_templates/index.html.haml]: Append `.` at the end of the title or name project :)

Doc: [succeed](http://www.rubydoc.info/github/haml/haml/Haml%2FHelpers%3Asucceed)

Comment (ChrisBr) [inline src/api/app/views/webui/image_templates/index.html.haml]: Nice :+1:

**Comment by ChrisBr** [comment]
LGTM

**Pull request merged**